chromadb/test/test_cli.py

```diff
@@ -0,0 +1,21 @@
+from typer.testing import CliRunner
+
+from chromadb.cli.cli import app
+
+runner = CliRunner()
+
+
+def test_app() -> None:
+    result = runner.invoke(
```

Comment:
Do we need to kill the spawned process? Not sure how runner works

Reply:
the test flag skips spawning the process https://github.com/chroma-core/chroma/pull/1032/files/135b997ba746a6cdb2ecad103307ddc84343d5b7#diff-a43650cef3ceef09cabaa56762839e49a8a440115910f6366216507237b9a3daR66

it just checks to make sure the flags made it in correctly